### DISCUSSION TITLE
Describe what telemetry would be collected

### DISCUSSION THREAD

**Discussion by Mart-Bogdan** [spacedriveapp/spacedrive#1533] [Feature Request]
![image](https://github.com/spacedriveapp/spacedrive/assets/775038/82497db0-07d5-44e2-b60d-d5fe0f4792be)

I'm not against telemetry, but I want to know what exactly is collected

And this should be described directly on e screen that asks you to opt in

**Comment by PineappleRind** [comment]
agree that it should be shown during onboarding — if not in text, then maybe an info circle or "Learn more"?

**Comment by brxken128** [comment]
Only pings every 4 and a half minutes are the "bare minimum", as that is just under what timeframe Plausible consider a user to be active.

In relation to the `pageView` event and the path: the path contains absolutely nothing exclusive to any individual user - not even library UUIDs (an example is `/:libraryId/overview`, more examples can be seen in the provided image).

<img width="182" alt="image" src="https://github.com/spacedriveapp/spacedrive/assets/77554505/3aaee854-e7ab-4ad6-996d-ab39c6230a25">

We do not log the actual paths being viewed on your system, but the generic path provided by the app's router (with any dynamic elements being replaced with their component name, such as `:id` or `:libraryId`.

Enabling full telemetry means that all of the pageview/addition/deletion/assigning events are sent, but again they contain **zero** user-identifiable information and absolutely nothing specific to any individual (it's essentially just a counter of how many libraries/locations/etc are created and deleted).

We also receive: whether or not the app is in development mode, the current core version, the current commit hash (could help narrow down bugs if the app and core ever go out-of-sync), whether or not the user opted to provide "full"/all telemetry, and which platform is being used (which at the moment (for production builds) is always `desktop`, as that's all we have officially released. The other option here is `web`, but that'll come into play once we have web and server options released (or while someone is running `web` in debug mode)).

We use [Plausible](https://plausible.io/) to receive telemetry, and they are a privacy-friendly, EU hosted provider who also collect no PII. They are [open source](https://plausible.io/open-source-website-analytics), self-hostable, and you may find their privacy policy [here](https://plausible.io/privacy). They even provide a [live demo](https://plausible.io/plausible.io), which gives you an idea of what we receive on our end.

I will open an issue and get this information added to our documentation, with a link available via a tooltip during both onboarding and the settings page in-app as these are where you are able to configure telemetry sharing settings.
